Ms/new text shadow

martensievers opened the pull request:
When adding text-shadow to the view, I first check if a single string as text_shadow attribute exists to support the newly added text_shadow property from the Incito format, before making sure to render existing text shadows to support backwards compatibility.

Commit:
feature: add metadata with title to view

Commit:
feat use new text_shadow attribute if provided

Commit:
refactor remove falsly added meta title on View

mortenbo commented on lib/coffeescript/views/text.coffee:
```diff
@@ -66,7 +66,9 @@ export default class TextView extends View
         # Text shadow.
         textShadow = @getTextShadow()
```
What about this line?

martensievers replied:
I think we still need to check for the old text_shadow to make sure this doesn't break backwards, so I left in the old way and use it as a fallback.

mortenbo commented on .idea/workspace.xml:
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectId" id="1at6GbNNng2Brbsdcw2MMIMFR5A" />
```
Can you add .idea/ to .gitignore?

mortenbo replied:
Or git rm as I can see it's already there.

martensievers replied:
Sure, will do.

Commit:
remove .idea/ files

Commit:
feat 🎸 add new text-shadow property to incito format

The new text-shadow property adds support for composite shadows by concatenating text-shadows in x,y,r,hex values separated by commas

martensievers commented:
The new text-shadow property adds support for composite shadows by concatenating text-shadows in x,y,r,hex values separated by commas

Commit:
Merge branch 'develop' into ms/new-text-shadow
